data/CONTRIBUTORS.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
ya no clue why this file name is capitalized

Reply:
wait why does this file exist

Reply:
no no no no no just use the github api please so much easier

Reply:
I thought we agreed to use dummy data until MVP stage passes and we can optimize stuff.

Reply:
i agree with dummy data for the projects but for the contributors?? unnecessary, it's just a simple fetch and render

Reply:
Fair point. I’ll resolve this tomorrow.